conversation/frontend/src/hooks/useMessageReplyOrTransfer.tsx

```diff
@@ -0,0 +1,118 @@
+import { useDate, useEdificeClient } from '@edifice.io/react';
+import { Message, Recipients } from '~/models';
+import { DEFAULT_MESSAGE, useMessage } from '~/services';
+import { useI18n } from './useI18n';
+
+export interface MessageReplyOrTransferProps {
```

Comment:
Le typage n'est pas clair. A quoi servent tous ces IDs ?
S'il sont exclusifs (c.à.d. 1 seul ID pris en compte dans la logique du hook) alors autant typer comme ça : 
```suggestion
export interface MessageReplyOrTransferProps {
  messageId: string | undefined;
  action: "reply" | "replyAll" | "transfer";
}
```